lib/signs_ui/config/state.ex

```diff
@@ -71,16 +72,22 @@ defmodule SignsUi.Config.State do
 
   @spec init(any()) :: {:ok, t()} | {:stop, any()}
   def init(_) do
-    case Config.Request.get_state() do
-      {:ok, config} ->
-        # re-save state, since format was updated
-        save_state(config)
-        schedule_clean(self(), 60_000)
-        {:ok, config}
+    schedule_clean(self(), 60_000)
+    config_store = Application.get_env(:signs_ui, :config_store)
+    response = config_store.read() |> Jason.decode!()
+
+    state = %{
+      signs:
+        Map.get(response, "signs", %{})
```

Comment:
Suggestion based on our slack convo: this is one spot where I might prefer to avoid the single pipe
```
response
|> Map.get("signs", %{})
|> Map.new(fn {sign_id, config} -> {sign_id, Sign.from_json(sign_id, config)} end)
```
for example just looks a little neater to me. But up to you if you like it as is